packages/mcp/package.json

```diff
@@ -82,8 +82,7 @@
   "wireit": {
     "build": {
       "dependencies": [
-        "compile",
-        "lint"
```

Comment:
decouple `lint` from the compilation script.

This is a convention we follow in our other CLI/libraries repo where `yarn build` does compile + lint but in the MCP monorepo we have those 2 separate:
https://github.com/salesforcecli/mcp/blob/d246f4544efdd007d4f56c64a0ed61d0e0d7ae1c/package.json#L25